### DISCUSSION TITLE
[Feature] Sort upload priority by photos first and then videos

### DISCUSSION THREAD

**Discussion by klejejs** [immich-app/immich#3846] [Feature Request]
### The feature

Large videos take a long to upload which basically breaks all background upload functionality (iOS), as they never finish uploading and always retry. To fix this, photos could be uploaded first and videos after, just like GPhotos does.

### Platform

- [ ] Server
- [ ] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #3341
